examples/helloworld/client.ts

```diff
@@ -99,9 +101,9 @@ export const APP_SPEC: AppSpec = {
   }
 }
 
-export type CallRequest<TSignature extends string, TArgs = undefined> = {
+export type CallRequest<TSignature extends string | undefined> = {
```

Comment:
This type probably needs a better name - CallRequest was only ever a placeholder. I guess it's just call args, but that's already taken. `TypedCallArgs`?

Reply:
@robdmoore ??